packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -48,7 +51,22 @@ export type StartRum<C extends RumInitConfiguration = RumInitConfiguration> = (
 
 type StartRumResult = ReturnType<typeof startRum>
 
-export function makeRumPublicApi<C extends RumInitConfiguration>(startRumImpl: StartRum<C>) {
+export interface RecorderApi {
+  public: {
+    startSessionReplayRecording: () => void
+    stopSessionReplayRecording: () => void
+  }
+  onRumStart: (
```

Comment:
Nesting those APIs in a `public` object seems odd to me.
Any reason to promote it that much?

Reply:
I made it so it is clear that we need to wrap those functions in `monitor`, and we can just [merge](https://github.com/DataDog/browser-sdk/pull/941/files#diff-f16464c555cdc03f7a8b03f1b312bb09ea41cd1acf4fa773629e880a5a1ea72bR217) it in the rest of the API.

Reply:
yep, by not doing the merge, we could:
- let `rumPublicApi` wrap these functions in monitor
- simplify the naming in `recordingApi` and use the public name only on `rumPublicApi` side:
```
rum.startSessionReplayRecording()
recording.start()
```
wdyt?

Reply:
Sounds good to me